contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -556,9 +556,8 @@ def _execute(self, number_of_f5_presses: int):
 
         self._stop_capture_if_necessary(capture_tab)
 
-        wait_for_condition(lambda: self.find_control(
-            'Window', 'Finalizing capture', recurse=False, raise_on_failure=False) is None,
-                           max_seconds=120)
+        capture_options_button = self.find_control('Button', 'Capture Options', parent=capture_tab)
+        wait_for_condition(lambda: capture_options_button.is_enabled() is True, max_seconds=30)
```

Comment:
Thanks a bunch for looking into this! 

Do you understand why this works and waiting for the 'Finalizing capture' does not?

Reply:
We had the theory that it failed because 'Finalizing capture' dialog appeared after the wait_for_condition.

Reply:
As discussed offline, lgtm.